robot.py

```diff
@@ -7,7 +7,7 @@
 from components.drivetrain import Drivetrain
 from components.bunnygrabber import BunnyGrabber
 from components.flipper import Flipper
-
+#these are components that have been coded in the components folder
```

Comment:
We don't need to document how python imports work. Python imports are already documented here: https://docs.python.org/3/reference/simple_stmts.html#import